code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -45,23 +45,30 @@
 	user.Stun(20 SECONDS)
 	user.forceMove(src)
 	occupant = user
-	update_icon(UPDATE_OVERLAYS)
+	update_icon(UPDATE_OVERLAYS | UPDATE_ICON_STATE)
 	feedinTopanim()
 	addtimer(CALLBACK(src, PROC_REF(startgibbing), user), 33)
 	return OBLITERATION
 
+/obj/machinery/gibber/update_icon_state()
+	. = ..()
```

Comment:
```suggestion
```
Has no use to call the parent here right?

Reply:
Ah, no I was mixing it up with `update_icon`